oauth2_provider/checks.py

```diff
@@ -26,3 +26,16 @@ def validate_token_configuration(app_configs, **kwargs):
         return [checks.Error("The token models are expected to be stored in the same database.")]
 
     return []
+
+
+@checks.register()
+def validate_backchannel_logout(app_configs, **kwargs):
+    errors = []
+
+    if oauth2_settings.OIDC_BACKCHANNEL_LOGOUT_ENABLED:
+        if not callable(oauth2_settings.OIDC_BACKCHANNEL_LOGOUT_HANDLER):
```

Comment:
OIDC_BACKCHANNEL_LOGOUT_HANDLER is not documented. What are the use cases for making a setting to override the logout handler? Is the specification flexible enough that it makes sense to give people the option to override this?

Reply:
Delegating to celery sounds reasonable, especially if people want to implement retries etc. Do we not have logging for error tracking or are you thinking of some particular 3rd party instrumentation? Can you expand on your thoughts in regard to error tracking?